contrib/backup/multicloud/auth/keystone.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2019 The OpenSDS Authors.
```

Comment:
client has the same feature code, please use that one.

Reply:
sorry，I can not  use that, because the code is not same, and function is not the same.